### ISSUE TITLE
JsonMappingException for PlayerAttack.vehicle and PlayerTakeDamage.attacker

### ISSUE THREAD

**Issue by Tetraquark** [kevcodez/pubg-api-kotlin#10]
At version 0.3.0 got `JsonMappingException` exceptions while getting telemetry from ApiClient:

for `PlayerAttack.vehicle` field:
```
com.fasterxml.jackson.databind.JsonMappingException: Parameter specified as non-null is null: method de.kevcodez.pubg.model.telemetry.events.PlayerAttack.setVehicle, parameter <set-?>
...
at de.kevcodez.pubg.client.ApiClient.getTelemetryData(ApiClient.kt:105) ~[pubg-api-wrapper-0.3.0.jar:na]
```
and for `PlayerTakeDamage.attacker` field:
```
com.fasterxml.jackson.databind.JsonMappingException: Parameter specified as non-null is null: method de.kevcodez.pubg.model.telemetry.events.PlayerTakeDamage.setAttacker, parameter <set-?>
...
at de.kevcodez.pubg.client.ApiClient.getTelemetryData(ApiClient.kt:105) ~[pubg-api-wrapper-0.3.0.jar:na]
```

These class fields are declared as lateinit not null var. But it looks like these fields may be null.

**Comment by kevcodez** [comment]
Hey @Tetraquark,

thanks for reporting the issue. Would you mind posting the URL to the telemetry json here so I can look into it?

**Comment by Tetraquark** [comment]
Yep, here it is: https://telemetry-cdn.playbattlegrounds.com/bluehole-pubg/pc-ru/2018/07/14/17/47/0bd72ef1-878e-11e8-a63e-0a586461bf1e-telemetry.json

**Comment by kevcodez** [comment]
Fields are optional now. Released 0.3.1, should be on maven central within 10 minutes.

I really need to do a proper integration test to avoid such issues.

Again, thanks for reporting ;)
